tests/test_dataset_signatures.py

```diff
@@ -0,0 +1,53 @@
+from inspect import signature
+
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+import reciprocalspaceship as rs
+
+
+def test_reset_index_dataseries():
+    """
+    Minimal example from GH#223
+    """
+    result = rs.DataSeries(range(10)).reset_index()
+    expected = pd.Series(range(10)).reset_index()
+    expected = rs.DataSet(expected)
+    assert_frame_equal(result, expected)
+
+
+def test_reset_index_signature(dataset_hkl):
```

Comment:
This looks like a good way to test whether pandas changed. Maybe we should do this test for more functions.

Reply:
The only other methods we directly overload and call `super()` on use the *args, **kwargs paradigm to overload, so I didn't bother adding the tests. I'm open to adding such methods, but it didn't seem like such a test would actually be informative (because they directly pass through arguments).

Reply:
cool. sounds like a non-issue. still a good trick to keep in mind.